demo/app/main-page-android.xml

```diff
@@ -0,0 +1,11 @@
+<Page xmlns="http://schemas.nativescript.org/tns.xsd" loaded="pageLoaded">
+  <ActionBar title="Loading Indicator Demo"/>
+  <GridLayout rows="*,auto">
+    <ScrollView>
+      <StackLayout row="0">
+        <Button text="Message" tap="{{showLoader}}" />
```

Comment:
Could we have a space either side so this would read `{{ showLoader }}` please? Likewise for the other bindings :smile:.